### ISSUE TITLE
GLPI 9.4 Compatibility

### ISSUE THREAD

**Issue by kornelson** [pluginsGLPI/room#36]
Please, would you update the plugin to support GLPI 9.4 Thanks. Is a very usefull plugin. thanks.

**Comment by ShkerzyA** [comment]
Yeah, would you update your very usefull and important plugin to support GLPI 9.4. Many thanks to You

**Comment by reinethernal** [comment]
Indeed. Compatibility with 9.4 is really nessesary

**Comment by jackburton79** [comment]
I installed on glpi 9.4.2 the alpha version 3.1.2, modified line 58 of rooms/setup.php from
  if (version_compare(GLPI_VERSION, '9.3.1', '>=') && version_compare(GLPI_VERSION, '9.4', '<')) {
to 
  if (version_compare(GLPI_VERSION, '9.3.1', '>=') && version_compare(GLPI_VERSION, '9.5', '<')) {

and was able to use the plugin on that version. I haven't done extensive testing, yet, but it seems to work
